apps/odyssey-stats/src/styles/typography.scss

```diff
@@ -0,0 +1,21 @@
+@import "@automattic/components/src/styles/typography.scss";
+
+$display-type-font: -apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen-Sans","Ubuntu","Cantarell","Helvetica Neue",sans-serif;
+$display-type-weight: 700;
+$display-type-line-height: 40px;
+
+// Override display type styling for Jetpack.
+@mixin stats-section-header-for-jetpack {
```

Comment:
I'm not an expert of SCSS, just wondering - are we able to use some kind of env variable (e.g. $is-jetpack ?) so that we only compile the Emrald typography conditionally, so that we don't have to override anymore.

Reply:
If our styling codebase were more centralized, we'd be able to leverage [the `!default` flag for variables](https://sass-lang.com/documentation/variables#default-values) and sort out this problem by overriding that default variable. This is how a lot of SCSS frameworks used to provide theming support back in the day.

There's no canonical way to perform environmental variable detection in a built-in way with SCSS. We can absolutely [hack something up like this](https://stackoverflow.com/questions/60058352/pass-webpack-environment-variable-to-scss-file), but that'll make things even harder to maintain than what I'm proposing, in my opinion.